### PR TITLE
fix: the explanation icon overlaps with a string

### LINKED ISSUE #476: [FIX]: The Explanation icon overlaps with a string
## Describe the bug
https://infermedica.atlassian.net/browse/MGPFE-220?atlOrigin=eyJpIjoiYThjZGQwMDUxNmUxNGJiODg3ZmNhNjg2ZjkzMTM0NWMiLCJwIjoiaiJ9

## Screenshots
Screenshot 2024-05-09 at 13 30 48
Screenshot 2024-05-09 at 13 32 46

### PR DESCRIPTION
**Pull request by pspaczek**
## Description
UiMultipleAnswareItem can't get UiListItemSuffix template refs to get it size.

## Related Issue
Closes #476 

## Screenshots (if appropriate):
Zrzut ekranu 2024-05-10 o 11 28 34
Zrzut ekranu 2024-05-10 o 11 28 59

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue [hunk 1]
```diff
@@ -188,16 +189,26 @@ const errorClass = computed(() => (props.invalid
...
+  }
+  return {
+    '--_label-suffix-width': `${width}px`,
+    '--_label-suffix-height': `${height}px`,
+  };
 });
 const handleInfoFocus = (event: KeyboardEvent) => {
   if (event.key !== 'ArrowRight') return;
-  if (suffix.value?.$el) {
+  if (suffixEl.value) {
```
Comment (AlanLes) [inline src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue:209]:
```suggestion
  if (!suffixEl.value) return;
```
and then without indentation, wdyt? :)

File: src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue [hunk 2]
```diff
@@ -188,16 +189,26 @@ const errorClass = computed(() => (props.invalid
...
+  let height = 0;
+  if (suffixEl.value) {
+    const size = suffixEl.value.getBoundingClientRect();
+    width = size.width;
+    height = size.height;
+  }
+  return {
+    '--_label-suffix-width': `${width}px`,
+    '--_label-suffix-height': `${height}px`,
+  };
```
Comment (AlanLes) [inline src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue:205]: Isn't it shorter and simpler? 😄 
```suggestion
const suffixSize = computed(() => {
  const size = suffixEl.value.getBoundingClientRect();
  const width = suffixEl.value ? size.width : 0;
  const height = suffixEl.value ? size.height : 0;
  return {
    '--_label-suffix-width': `${width}px`,
    '--_label-suffix-height': `${height}px`,
  };
```

**Review by AlanLes — APPROVED** [review]
lgtm 👍

**Comment by pspaczek** [comment]
@AlanLes your suggestions are included. Can you reapprove this PR?

**Review by jbucior — APPROVED** [review]
LGTM

**Review by AlanLes — COMMENTED** [review]
Overall looks good, but... I mentioned it in one of the other PRs: why are those ref `listItemSuffixTemplateRefs`, `suffixTemplateRefs` variables called like `...Refs` which suggest that it's a _**plural**_ when, in fact, it isn't?

**Comment by pspaczek** [comment]
> Overall looks good, but... I mentioned it in one of the other PRs: why are those ref `listItemSuffixTemplateRefs`, `suffixTemplateRefs` variables called like `...Refs` which suggest that it's a _**plural**_ when, in fact, it isn't?

renamed

**Review by DanielHajduk98 — APPROVED** [review]

**Review by AlanLes — APPROVED** [review]

**Review by kubaszajna — APPROVED** [review]

**Pull request merged**